zerver/lib/slack_data_to_zulip_data.py

```diff
@@ -465,5 +526,4 @@ def do_convert_data(slack_zip_file: str, realm_subdomain: str, output_dir: str)
 
     print('######### DATA CONVERSION FINISHED #########\n')
     print("Zulip data dump created at %s" % (output_dir))
-    print("Import Command: ./manage.py import --destroy-rebuild-database %s\n" % (output_dir))
```

Comment:
Should the documentation be updated too?

Reply:
https://github.com/zulip/zulip/blob/master/templates/zerver/help/import-data-from-slack.md it already includes both `import-from-nonempty` and `destroy-rebuild-database`. I'll be thoroughly updating it after the tests have been added.